gluonnlp/criterion/language_model.py

```diff
@@ -0,0 +1,123 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=
+"""Language model criterion."""
+
+__all__ = ['ActivationRegularizationLoss', 'TemporalActivationRegularizationLoss']
+
+from mxnet import nd
+
+class ActivationRegularizationLoss(object):
```

Comment:
inherit mx.gluon.loss.Loss

Reply:
examples https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/loss.py

Reply:
The inputs to both losses are list. It is straightforward to use object.